pages/projects/[id]/tasks/[taskId]/index.js

```diff
@@ -55,14 +56,29 @@ export default function TaskDetailPage() {
 
   const date = new Date(createdAt).toLocaleDateString();
 
-  async function handleDelete() {
-    await fetch(`/api/projects/${id}/tasks/${taskId}`, {
-      method: "DELETE",
+  function handleDelete() {
+    Swal.fire({
+      icon: "question",
+      iconColor: "#d1cffd",
+      title: "Are you sure?",
+      text: "Do you really want to delete this task? ",
+      type: "question",
+      showCancelButton: true,
+      confirmButtonColor: "#d1cffd",
+      confirmButtonText: "Delete",
+      closeOnConfirm: true,
+    }).then(async (result) => {
+      if (result.isConfirmed) {
+        await fetch(`/api/projects/${id}/tasks/${taskId}`, {
+          method: "DELETE",
+        });
+        mutate(`api/projects/${id}`);
```

Comment:
We were wondering why you use "mutate" here but not in the other handleDelete function (in the projects-file(s)). Is it necessary to have it here?

Reply:
Hey Roland. thanks for the comment! we have edited it.